model/jpa/src/main/resources/META-INF/jpa-changelog-master.xml

```diff
@@ -86,5 +86,6 @@
     <include file="META-INF/jpa-changelog-26.0.0.xml"/>
     <include file="META-INF/jpa-changelog-26.1.0.xml"/>
     <include file="META-INF/jpa-changelog-26.2.0.xml"/>
+    <include file="META-INF/jpa-changelog-26.2.1.xml"/>
```

Comment:
With this being an enhancement, and while we don't backport enhancemnets, this should then be 26.3 and not a patch release. We also avoid updating databases in patch releases, as it could lead to downtimes when applied to large tables. While this might be ok for minor releases, this is unexpected for patch releases.

```suggestion
    <include file="META-INF/jpa-changelog-26.2.1.xml"/>
```